.github/workflows/e2e-test-apply.yml

```diff
@@ -72,6 +91,14 @@ jobs:
           RegionID: ${{ secrets.RegionID }}
         if: ${{ github.event.comment.body == '/test apply' || github.event.comment.body == '/test all' || github.event_name == 'push' }}
         run: |
-          # fix:open /proc/sys/net/netfilter/nf_conntrack_max: permission denied, see: https://github.com/kubernetes-sigs/kind/issues/2240
+          # fix bug in kernal 5.12.2+:open /proc/sys/net/netfilter/nf_conntrack_max: permission denied, see: https://github.com/kubernetes-sigs/kind/issues/2240
           sudo sysctl net/netfilter/nf_conntrack_max=131072
           ginkgo -v --focus="sealer apply" test
+      
+      - name: Set final commit status
```

Comment:
Is there any conflict in setting status for multiple tasks concurrently?

Reply:
No, It only change the final status to the right value(fail or success).